drivers/stepper/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 # SPDX-FileCopyrightText: Copyright (c) 2023 Carl Zeiss Meditec AG
 # SPDX-License-Identifier: Apache-2.0
 
+add_subdirectory_ifdef(CONFIG_GPIO_STEPPER gpio_stepper)
```

Comment:
why adding subdirectories? most driver classes just have a single plain folder

Reply:
I am following the pattern as with sensors drivers

Reply:
The sensor subsystem is a bit of a special case, I would not bother with those at this stage.

Reply:
@fabiobaltieri So you suggest to put all the drivers in a single plain folder as well?

Reply:
yes that's a better starting point for single file drivers

Reply:
done